tests/test_remote_datasets.yml

```diff
@@ -0,0 +1,8 @@
+dataset_1:
+  url: http://not-a-real-url.com/dataset_1.tar
+  files: 
+    - file_1.root
+dataset_2:
```

Comment:
Do you not want to add an example with 2 files?

Reply:
Yeah, that could be sensible.  If I'm honest I'm not really testing fully the remote files functions at this point, I'm not sure how best to test things that need to interact with a remote server.

Reply:
Indeed tricky.